node/node.go

```diff
@@ -87,6 +87,12 @@ type Node struct {
 	pprofSrv          *http.Server
 }
 
+type mempoolReactor interface {
```

Comment:
Should we define this in the mempool package?

Reply:
Following the service example, maybe `BasicMempoolReactor`